modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portal/_accessibility.scss

```diff
@@ -0,0 +1,23 @@
+.sr-only {
+	border: 0;
+	clip: rect(0, 0, 0, 0);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	white-space: nowrap;
+	width: 1px;
+}
+
+.sr-only-focusable {
+	&:active,
+	&:focus {
+		clip: auto;
+		height: auto;
+		overflow: visible;
+		position: static;
+		white-space: normal;
+		width: auto;
+	}
+}
```

Comment:
Interesting to see how it is implemented. I just used it 😅

Reply:
haha, copy / pasted from BS4

Reply:
I can't see any error in this code, but I have concerns about the change itself. A few months ago we made an effort to remove as many classes as possible from `frontend-theme-styled/../main.css` now it seems we want to move some of them back. I understand it's necessary but I would like to explore, a little bit more, some alternative solutions before agreeing to this.
- The first one would be using the [`!default` flag from SASS](https://sass-lang.com/documentation/variables#default-values) but it requires a common .scss file as base for both output `clay.css` and `main.css`. And even in that case I'm not sure how to make the classes in `main.css` conditionals depending on `clay.css`.
- The second one is more promising, using the [CSS Custom Properties as conditionals](https://dev.to/siddharthshyniben/conditional-logic-with-css-the-css-custom-property-trick-44hb), but also in this case we would be creating a duplication of the class in the `main.css` that doesn't work and breaks the `clay.css` one.

/cc @pat270 

---

Edit: this PR has been forwarded while I was reviewing it 😞 – it's ok, we can apply a new solution if we find it!

Reply:
I don't think it's that big a deal to duplicate it. We can check if a unique Clay function exists in Sass with `function-exists()`. If it exists we can assume that Clay is being loaded and not output this CSS.

Reply:
doesn't it require an unified output? I mean, how its file architecture/imports would be? _clay_variables.scss => clay.css => main.css?

Reply:
@edalgrin oh yeah, we can't assume clay.css is fully loaded based on an existing clay function in main.scss. I was assuming if someone didn't want Clay, they would also remove it from main.scss. I still think we should duplicate it like we are doing in this pr.

Reply:
Alright, I don't want to hold this anymore, so LGTM! (and hope we find a better solution in the future 😁)